doc/source/conf.py

```diff
@@ -55,7 +55,7 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'Conduce Python API'
```

Comment:
I don't think we need both changes.  I'd like to see what it looks like without this change.